src/plugins/login/azureLoginPlugin.test.ts

```diff
@@ -68,7 +68,7 @@ describe("Login Plugin", () => {
       undefined // would be options
     )
     expect(AzureLoginService.interactiveLogin).not.toBeCalled();
-    expect(sls.variables["azureCredentials"]).toEqual(credentials);
+    expect(JSON.stringify(sls.variables["azureCredentials"])).toEqual(JSON.stringify(credentials));
```

Comment:
Any reason you are comparing the serialized string vs. the object using toEqual()?

Reply:
I had an issue comparing the objects directly, jest said that they would serialize to the same thing, but did give more details

Reply:
I had an issue comparing the objects directly, jest said that they would serialize to the same thing, but did give more details